docs/getting-started.rst

```diff
@@ -1,59 +1,106 @@
+---------------
 Getting started
 ---------------
 
-Requirements:
 
-- ``docker`` installed and running
-- ``Node.js`` >= 7.6 - ``async/await`` support
 
-1. Download repositories: gardener-server, gardener-smart-contracts
+
+Repositories
+------------
+
+Gardener consists of three repositories, two main ones: gardener-smart-contracts which holds smart contracts, and gardener-server, which is responsible for fetching data from third party data sources.
+The third one, gardener-monitor is optional and it helps visualizing requests.
+
+1. Gardener server
+::
+
+  git clone https://github.com/EspeoBlockchain/gardener-server.git
+
+2. Gardener smart contracts
+::
+
+  git clone https://github.com/EspeoBlockchain/gardener-smart-contracts.git
+
+3. Gardener monitor (optional)
+::
+
+  git clone https://github.com/EspeoBlockchain/gardener-monitor.git
+
+
+Running blockchain - Server
```

Comment:
```suggestion
Running the test blockchain - Ganache
```
I'm not sure these "Running blockchain - XYZ" headings really tell the story of the contents (applies to other headings as well).